codesign/README.md

```diff
@@ -0,0 +1,9 @@
+A standalone app to codesign Mac engine binaries.
```

Comment:
App -> tool

I think the following sentences can be removed, and for now a single sentence is fine.